collections/misc/commentlist.php

```diff
@@ -156,54 +201,11 @@
-								<?php
-								$userArr = $commentManager->getCommentUsers($showAllGeneralObservations);
-								foreach($userArr as $userid => $userStr){
-									echo '<option value="'.$userid.'" '.($uid==$userid?'selected':'').'>'.$userStr.'</option>';
-								}
-								?>
-							</select>
-						</div>
-						<?php
-						if($IS_ADMIN && $collMeta['colltype'] == 'General Observations'){
-							echo '<div><input name="showallgenobs" type="checkbox" value="1" onchange="this.form.submit()" '.($showAllGeneralObservations?'checked':'').' /> '.$LANG['DISP_ALL_GEN_OBS'].'</div>';
-						}
-						?>
-						<div>
-							<label for="tsstart"><?php echo $LANG['DATE']; ?>: </label>
-							<input id="tsstart" name="tsstart" type="date" value="<?php echo $tsStart; ?>" title="<?php echo (isset($LANG['START_DATE']) ? $LANG['START_DATE'] : 'Start Date'); ?>" aria-label="<?php echo (isset($LANG['START_DATE']) ? $LANG['START_DATE'] : 'Start Date'); ?>"/>
-							- <input name="tsend" type="date" value="<?php echo $tsEnd; ?>" title="<?php echo (isset($LANG['END_DATE']) ? $LANG['END_DATE'] : 'End Date'); ?>" aria-label="<?php echo (isset($LANG['END_DATE']) ? $LANG['END_DATE'] : 'End Date'); ?>" />
-						</div>
-						<fieldset>
-							<legend> <?php echo (isset($LANG['COMMENT_TYPE']) ? $LANG['COMMENT_TYPE'] : 'Comment Type'); ?> </legend>
-							<input id="public" name="rs" type="radio" value="1" <?php echo ($rs==1?'checked':''); ?> /> <label for="public"> <?php echo $LANG['PUBLIC']; ?> <br/> </label>
-							<input id="nonpublic" name="rs" type="radio" value="2" <?php echo ($rs==2?'checked':''); ?> /> <label for="nonpublic"> <?php echo $LANG['NON-PUBLIC']; ?> <br/> </label>
-							<input id="reviewed" name="rs" type="radio" value="3" <?php echo ($rs==3?'checked':''); ?> /> <label for="reviewed"> <?php echo $LANG['REVIEWED']; ?> <br/> </label>
-							<input id="all" name="rs" type="radio" value="0" <?php echo (!$rs?'checked':''); ?> /> <label for="all"> <?php echo $LANG['ALL']; ?> </label>
-						</fieldset>
-						<div class="top-breathing-room-rel" >
-							<button type="submit" name="submitbutton" value="Refresh List"><?php echo $LANG['REFRESH_LIST']; ?></button>
-						</div>
-						<div>
-							<input name="collid" type="hidden" value="<?php echo $collid; ?>" />
-						</div>
-					</form>
-				</section>
 				<?php
 				if($commentArr){
 					foreach($commentArr as $comid => $cArr){
 						echo '<div style="margin:15px;">';
-						echo '<div style="margin-bottom:10px;"><a href="../individual/index.php?occid=' . htmlspecialchars($cArr['occid'], HTML_SPECIAL_CHARS_FLAGS) . '" target="_blank">' . htmlspecialchars($cArr['occurstr'], HTML_SPECIAL_CHARS_FLAGS) . '</a></div>';
+						echo '<div style="margin-bottom:10px;"><a href="../individual/index.php?occid=' . htmlspecialchars($cArr['occid'], HTML_SPECIAL_CHARS_FLAGS) . '" target="_blank"><b>' . strip_tags($cArr['occurstr']) . '</b></a></div>';
```

Comment:
I'm curious why you opted for strip_tags() here. I don't think it matters overly much, because I doubt we'll want to render any html-specific characters here. Just curious.

Reply:
That seemed the best choice given that $cArr['occurstr'] contains several separated elements ( bold tag, span tag, plain text). By stripping of tags I turn it into one line of text which is better represented as link on user display:

Compare without strip tags:
<img width="309" alt="Screenshot 2024-03-01 at 12 21 15 PM" src="https://github.com/BioKIC/Symbiota/assets/86389284/e17b107f-de03-40b2-9c3a-6e435888ef95">

With strip tags:
<img width="309" alt="Screenshot 2024-03-01 at 12 22 03 PM" src="https://github.com/BioKIC/Symbiota/assets/86389284/f679e9a7-4367-4d13-8e26-958e3df9c54b">